scripts/covid_tracking_project/historic_state_data/config.txt

```diff
@@ -3,12 +3,12 @@ import: {
     provenance_url: "https://www.covidtracking.com"
     provenance_description: "COVID-19 related data for US states from The COVID Tracking Project."
     # Path to the resolved MCF.
-    mcf_url: "gs://resolved_mcf/external_tables/covid_tracking_states_fix/nodes.mcf-00000-of-00001"
+    mcf_url: "gs://resolved_mcf/external_tables/covid_tracking_states_new/nodes.mcf-00000-of-00001"
     # First date of imported data.
     start_date: "2020-01-22"
     # Last date of imported data.
-    end_date: "2020-06-22"
+    end_date: "2020-06-25"
     # Source data release schedule.
     update_frequency: "P1D"
-    next_release_date: "2020-06-23"
+    next_release_date: "2020-06-26"
```

Comment:
Can also remove end_date and next_release_date here so that future no need to keep updating this.

Reply:
Got it.